hash/gba.xml

```diff
@@ -35846,6 +35893,18 @@ Hangs during cutscene after beating Maxim for best ending path (i.e. before real
 		</part>
 	</software>
 
+	<software name="rainb6rsp" cloneof="rainb6rs">
+		<description>Tom Clancy's Rainbow Six - Rogue Spear (prototype, version 23, 20011025)</description>
+		<year>2002</year>
+		<publisher>Ubi Soft</publisher>
+		<part name="cart" interface="gba_cart">
+			<feature name="slot" value="gba_eeprom_4k" />
+			<dataarea name="rom" size="7963152">
+				<rom name="roguespear_v23.bin" size="7963152" crc="666e31b5" sha1="d8426022537149670871817ff9336261b31172b5"/>
+			</dataarea>
```

Comment:
This ROM size looks rather suspicious.  What was this actually dumped from?  Is it “trimmed”?  The GBA expects cartridges to behave like “burst ROM”, either with actual burs ROM chips, or using a counter and conventional ROM.  It’s usually a power-of-two size.